### DISCUSSION TITLE
Code Comments

### DISCUSSION THREAD

**Discussion by jgoll7** [jgoll7/Tafe22S211#10] [General]
Just a friendly reminder to make sure that you are adding comments to your code to make it easier to understand and navigate, most importantly using comments to explain the methods and validations.

**Comment by MoonlitBeans** [comment]
In both the C# and XAML code? Because I did it in the C# out of habit.

**Reply by jgoll7** [reply]
C# most definitely, XAML comments are nice if added but definitely looking for C# comments. C# code is definitely much easier to interpret with comments, good work!
